### PR TITLE
REVSDL-2380 HMI command factory for VR plugin

### PR DESCRIPTION
**Pull request by giangtqh**
@bonewell 
@DKlimenko 
@VladSemenyuk 
@KKarlash 
@thinhts 
please review

### CODE REVIEW

File: src/components/vr_cooperation/src/commands/hmi/active_service_request.cc [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+
+#include "vr_cooperation/commands/hmi/active_service_request.h"
+
```
Comment (bonewell) [inline src/components/vr_cooperation/src/commands/hmi/active_service_request.cc]: `
#include "utils/logger.h"
`

File: src/components/vr_cooperation/src/commands/hmi/process_data_request.cc [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+
+#include "vr_cooperation/commands/hmi/process_data_request.h"
+
```
Comment (bonewell) [inline src/components/vr_cooperation/src/commands/hmi/process_data_request.cc]: `
#include "utils/logger.h"
`

File: src/components/vr_cooperation/src/commands/hmi/request_from_hmi.cc [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+
+#include "vr_cooperation/commands/hmi/request_from_hmi.h"
+
```
Comment (bonewell) [inline src/components/vr_cooperation/src/commands/hmi/request_from_hmi.cc]: `
#include "utils/logger.h"
`

File: src/components/vr_cooperation/src/hmi_command_factory.cc [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+
+#include "vr_cooperation/hmi_command_factory.h"
+
+#include "vr_cooperation/commands/hmi/active_service_request.h"
+#include "vr_cooperation/commands/hmi/process_data_request.h"
+
```
Comment (bonewell) [inline src/components/vr_cooperation/src/hmi_command_factory.cc]: `
#include "utils/logger.h"
`

**Comment by bonewell** [comment]
Reviewed.

**Comment by giangtqh** [comment]
Now we don't need HMI command factory. So, closed (will not merge to SDLRPCService branch)

**Pull request closed without merging**